start-sources.sh

```diff
@@ -2,7 +2,7 @@ FULLPATH=$(echo $PWD)
 ALLYPATH=`dirname "${FULLPATH}"`
 
 ALLYCOM=${ALLYPATH}/components/
-PYTHONPATH=${PYTHONPATH}:${ALLYCOM}ally
```

Comment:
do it like this, better:
if [ -z "$LOCALPYTHONPATH" ]; then
   LOCALPYTHONPATH=${PYTHONPATH} 
fi
PYTHONPATH=${LOCALPYTHONPATH}